src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -507,16 +519,32 @@ fun sendInfotrygdOppdatering(
     itfh: InfotrygdForespAndHealthInformation,
     receivedSykmelding: ReceivedSykmelding,
     behandlerKode: String,
-    navKontorNr: String
+    navKontorNr: String,
+    jedis: Jedis
 ) {
     val perioder = itfh.healthInformation.aktivitet.periode.sortedBy { it.periodeFOMDato }
     val marshalledFellesformat = receivedSykmelding.fellesformat
     val personNrPasient = receivedSykmelding.personNrPasient
     val signaturDato = receivedSykmelding.sykmelding.signaturDato.toLocalDate()
     val tssid = receivedSykmelding.tssid
-    sendInfotrygdOppdateringMq(producer, session, createInfotrygdBlokk(marshalledFellesformat, itfh, perioder.first(), personNrPasient, signaturDato, behandlerKode, tssid, loggingMeta, navKontorNr), loggingMeta)
+    val sha256String = sha256hashstring(createInfotrygdBlokk(itfh, perioder.first(), personNrPasient, signaturDato, behandlerKode, tssid, loggingMeta, navKontorNr))
```

Comment:
Det er 5 brancher som kaller createInfotrygdBlokk direkte eller indirekte, men siden vi blander businesslogikk med transportobjektet så blir det veldig vanskelig å skrive det noe særlig bedre. 

For at denne appen skal være endringsdyktig så må vi på et eller annet tidspunkt skille mellom forretningslogikk og transportobjekter. Det er mye jobb og krever en del omskriving, men alternativet er at hver eneste PR dytter oss litt og litt lenger ned i gjørma. Det er vanskelig å gjøre samtidig som det "brenner" men vi _må_ finne ut hvordan vi skal komme oss der. For denne appen skal leve i noen år.

Reply:
Heilt enig med deg der.
Tenker at vi får mottaket opp å spinner skikkelig.
Fikser evt bugs and so on.
Får å så gjøre ein major refakotorening, som vil gjøre appen letter å leve med